### ISSUE TITLE
 Add a live demo once ref.detectChanges is merged into master --> 

### ISSUE THREAD

**Issue by piascikj** [piascikj/angular#96]
Issue created from code comment with imdone.io

----

TODO:  Add a live demo once ref.detectChanges is merged into master --> id:95
[**packages/core/src/change_detection/change_detector_ref.ts:61**](https://github.com/piascikj/angular/blob/b21a1d1961aa4039ee0d1479a0c6dc63c6ba8471/packages/core/src/change_detection/change_detector_ref.ts#L61)

@imdone - Efficiently manage your project's technical debt. [imdone.io](http://imdone.io)
